virttest/utils_net.py

```diff
@@ -1269,6 +1263,24 @@ def clean_ovs_env(run_dir="/var/run/openvswitch", selinux_mode=None,
         utils_misc.kill_process_by_pattern("ovs-vswitchd")
 
 
+def install_openvswitch():
+    """
+    Install openvswtich package
+
+    :return: True or False
+    """
+    if process.system("yum info openvswitch", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch")
+    elif process.system("yum info openvswitch2.11", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch2.11")
+    elif process.system("yum info openvswitch2.15", ignore_status=True) == 0:
+        utils_package.package_install("openvswitch2.15")
```

Comment:
We can't cover all versions of openvswitch2 in this function, someone told me they may have different ovs packages at the same time and user would select the proper version before the testing.
```
openvswitch2.11-2.11.3-67.el8fdp.ppc64le : Open vSwitch
Repo        : FDP
Matched from:
Filename    : /usr/bin/ovs-vsctl

openvswitch2.12-2.12.0-23.el8fdp.ppc64le : Open vSwitch
Repo        : FDP
Matched from:
Filename    : /usr/bin/ovs-vsctl

openvswitch2.13-2.13.0-58.el8fdp.ppc64le : Open vSwitch
Repo        : FDP
Matched from:
Filename    : /usr/bin/ovs-vsctl
```
So I think there can be 2 options:
1. make the version configurable, for instance, ` install_openvswitch(ovs_ver="2.*")`
2. install the correct package in ci, and check if ovs-vsctl cmd exists in codes.
What do you think? @yalzhang

Reply:
Thank you! Have add it to CI, it is cheaper, close this PR.